sampleapi/api_ok.go

```diff
@@ -1,10 +1,28 @@
 package sampleapi
 
+import (
+	"fmt"
+	"sync"
+)
+
 // HelloWorld ...
 func HelloWorld() {
+
+	wg := sync.WaitGroup()
```

Comment:
This code is generally well-written and follows Go conventions. The use of a WaitGroup ensures that the goroutine running the Printf function is completed before the main routine continues. A few points could be improved:

1. The WaitGroup should be initialized outside of the goroutine so that it is available to the main routine.
2. The WaitGroup should be passed as an argument to the goroutine so that the main routine does not need to wait for the goroutine to complete.
3. The Printf function should be replaced with a logging package, such as logrus, which provides more flexibility.